src/pages/scale.tsx

```diff
@@ -102,117 +102,146 @@ export function Scale({
             />
           </ButtonGroup>
         </Box>
-        <ZStack>
-          <Box
-            sx={{
-              display: 'flex',
-              width: '100%',
-              height: '100%',
-              borderRadius: 2
+        <div
+          style={{
+            flexGrow: 1,
+            display: 'grid',
+            gridTemplateRows: 'auto 1fr',
+            gap: 8
+          }}
+        >
+          <div
+            style={{
+              display: 'flex'
             }}
           >
             {scale.colors.map((_, i) => {
-              const color = getColor(palette.curves, scale, i)
-              const hex = colorToHex(color)
-              const contrast = focusedHex ? getContrast(hex, focusedHex) : undefined
-              const contrastScore = contrast ? getContrastScore(contrast) : undefined
               return (
-                <Box
+                <div
                   key={i}
-                  tabIndex={0}
-                  onFocus={() => setIndex(String(i))}
-                  sx={{
-                    outline: 'none',
+                  style={{
                     width: '100%',
-                    height: '100%',
-                    color: focusedHex,
-                    backgroundColor: hex,
-                    borderTopLeftRadius: i === 0 ? 2 : 0,
-                    borderBottomLeftRadius: i === 0 ? 2 : 0,
-                    borderTopRightRadius: i === scale.colors.length - 1 ? 2 : 0,
-                    borderBottomRightRadius: i === scale.colors.length - 1 ? 2 : 0,
-                    position: 'relative',
-                    fontSize: 1,
-                    display: 'grid',
-                    placeItems: 'end',
-                    p: 2,
-                    fontWeight: 'bold',
-                    '&::before': {
-                      content: '""',
-                      display: String(i) === index ? 'block' : 'none',
-                      position: 'absolute',
-                      top: '-8px',
-                      height: 4,
-                      left: 0,
-                      right: 0,
-                      backgroundColor: 'var(--color-text)',
-                      borderRadius: 2
-                    }
+                    textAlign: 'center',
+                    fontSize: 14
```

Comment:
Small design idea: What if we made the active index bolder?
```suggestion
                    fontSize: 14,
                    fontWeight: Number(index) === i ? 600 : 400
```